packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -547,6 +551,8 @@ abstract class ResidentRunner {
     // Listen for service protocol connection to close.
     for (FlutterDevice device in flutterDevices) {
       for (VMService service in device.vmServices) {
+        // Callback hookups' Futures are uninteresting.
+        // ignore: unawaited_futures
```

Comment:
same as earlier comment, though again, i don't understand why we're not awaiting these